pkgs/tools/misc/pferd/default.nix

```
@@ -5,14 +5,14 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "pferd";
-  version = "3.4.3";
+  version = "3.5.0";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "Garmelon";
     repo = "PFERD";
-    rev = "refs/tags/v${version}";
-    sha256 = "sha256-m9u4CLvyF2n61JQyxEKlVzWZCUhzVEVqpSZMNDZXCAo=";
+    rev = "v${version}";
+    sha256 = "1mi3ql6ryfgg31airq3kgz5a5lbprszddm0gggkrkwiz2cljlqw8";
```

Please use an SRI hash.
```suggestion
    hash = "1mi3ql6ryfgg31airq3kgz5a5lbprszddm0gggkrkwiz2cljlqw8";
```